### ISSUE TITLE
License

### ISSUE THREAD

**Issue by marcinczeczko** [ops4j/peaberry#84]
Hi,
Is there a particular reason why peaberry does not have LICENSE file in the github repository ?

**Comment by hwellmann** [comment]
Yes, it's sort of orphaned and unmaintained. The sources do have an Apache v2 copyright header, though.
